kiwi/volume_manager/base.py

```diff
@@ -64,33 +64,33 @@ def __init__(
         # all volumes are combined into one mountpoint. This is
         # needed at sync_data time. How to mount the volumes is
         # special to the volume management class
-        #: root mountpoint for volumes
+        # root mountpoint for volumes
         self.mountpoint: Optional[str] = None
 
-        #: dictionary of mapped DeviceProviders
+        # dictionary of mapped DeviceProviders
         self.device_map = device_map
 
         # bind the device providing class instance to this object.
         # This is done to guarantee the correct destructor order when
         # the device should be released.
         self.device_provider_root = device_map['root']
 
-        #: An indicator for the mount of the filesystem and its volumes
-        #: when mounted for the first time
+        # An indicator for the mount of the filesystem and its volumes
+        # when mounted for the first time
         self.volumes_mounted_initially = False
 
-        #: root directory path name
+        # root directory path name
         self.root_dir = root_dir
-        #: list of volumes from :class:`XMLState::get_volumes()`
+        # list of volumes from :class:`XMLState::get_volumes()`
         self.volumes = volumes
-        #: volume group name
+        # volume group name
         self.volume_group = None
-        #: map volume name to device node
+        # map volume name to device node
         self.volume_map: Dict[str, str] = {}
-        #: list of volume MountManager's
+        # list of volume MountManager's
         self.mount_list: List[MountManager] = []
 
-        #: main storage device node name
+        # main storage device node name
```

Comment:
Why are you removing the leading `:`? This makes Sphinx no longer recognize the comments as documentation.

Reply:
my fault, a check script removed them.... will fix it